cranelift/codegen/src/isa/s390x/inst/emit.rs

```diff
@@ -3216,7 +3216,7 @@ impl MachInstEmit for Inst {
                 }
 
                 let opcode = 0xc05; // BRASL
-                let reloc = Reloc::S390xPCRel32Dbl;
+                let reloc = Reloc::S390xPLTRel32Dbl;
```

Comment:
I think this should check is_colocated and use PCRel if true. That is faster than going through the PLT.